cheetah/utils/bmadx.py

```diff
@@ -1,5 +1,8 @@
+import numpy as np
 import torch
 
+double_precision_epsilon = np.finfo(np.float64).eps
```

Comment:
Is there no `torch` equivalent for this? I would also just pass this directly as the default argument rather than create a variable for it.

Reply:
I tried that but flake8 doesn't like passing directly a function call to default argument

Reply:
Ah ... I see ... this is probably not an issue here, but let's stay flake8-conform then and keep the variable.